ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -898,18 +910,22 @@ _ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR *self)
   TizenSensorSpec *spec;
   gchar *tensor;
   GstCaps *retval;
+  gchar *typestr = NULL;
 
   spec = g_hash_table_lookup (tizensensors, self->type);
 
   if (FALSE == self->configured || SENSOR_ALL == self->type || NULL == spec) {
     tensor = g_strdup_printf("other/tensor; other/tensors, num_tensors=1");
   } else {
-    tensor = g_strdup_printf("other/tensor, dimension=%u:%u:%u:%u ; "
-        "other/tensors, num_tensors=1, dimensions=%u:%u:%u:%u",
+    typestr = gst_tensor_get_type_string (spec->tinfo.type);
+    g_assert (typestr && typestr[0]);
+    tensor = g_strdup_printf("other/tensor, dimension=%u:%u:%u:%u, type=%s ; "
+        "other/tensors, num_tensors=1, dimensions=%u:%u:%u:%u, type=%s",
         spec->tinfo.dimension[0], spec->tinfo.dimension[1],
-        spec->tinfo.dimension[2], spec->tinfo.dimension[3],
+        spec->tinfo.dimension[2], spec->tinfo.dimension[3], typestr,
         spec->tinfo.dimension[0], spec->tinfo.dimension[1],
-        spec->tinfo.dimension[2], spec->tinfo.dimension[3]);
+        spec->tinfo.dimension[2], spec->tinfo.dimension[3], typestr);
+    g_free (typestr);
```

Comment:
gst_tensor_get_type_string() does not return newly allocated string

Reply:
Applied. Thanks!